zerocode-maven-archetype/README.txt

```diff
@@ -0,0 +1,22 @@
+To install archetype locally:
+    Navigate to zerocode-maven-archetype on your machine. Run "mvn install" from this directory
+    
+To generate anew archetype-based project:
+    Navigate to the directory that will house the project. Run 
+    "mvn archetype:generate 
+    -DarchetypeGroupId=com.myproject 
+    -DarchetypeArtifactId=zerocodeArchetype 
+    -DarchetypeVersion=1.0-SNAPSHOT 
+    -DgroupId=com.myproject 
+    -DartifactId=zerocodeArchetypeTest"
+    
+    The generic command format is:
+    "mvn archetype:generate -DarchetypeGroupId=<custom-archetype group id e.g.>
+    -DarchetypeArtifactId=<custom-archetype artifactid>
+    -DarchetypeVersion=<custom-archetype version>
+    -DgroupId=<new project Group Id>
+    -DartifactId=<new project artifact Id>"
+    
+Add personal GitHub token to test files:
+    In both post_api_200.json and put_api_200.json, substitute your own GitHub token for the placeholder.
+    In put_api_200.json, also update the name of the owner in the URL to your GitHub username
```

Comment:
@kristin-smith Do you want to prompt the user for custom properties so you can replace these in the Archetype without the user manually changing it?
The only downside would be the properties would need to be known at project generation time, I think for the username it would work quite well but on the fence about the GitHub bearer token

Example if you decide to implement - Below would go in the `archetype-metdata.xml` file:
```xml
<requiredProperties>
  
    <requiredProperty key="entityName"/>
    <requiredProperty key="gitUrl"/>
    <requiredProperty key="includeGetById"/>

  </requiredProperties>
```

Reply:
@jneate I'm actually not sure about implementing, just because this is really placeholder text. Would it be better to just put in comments instead describing what a typical Post or Put request might look like rather than getting hung up on getting an example to actually run?